testdata/configtest.json

```diff
@@ -12,6 +12,15 @@
  "profiler": {
   "enabled": false
  },
+ "ntpclient": {
+  "enabled": 0,
+  "pool": [
+   "0.pool.ntp.org:123",
+   "pool.ntp.org:123"
+  ],
+  "allowedDifference": 50000000,
+  "allowedNegativeDifference": 0
```

Comment:
Should this be zero, or even better revert all changes to this configtest.json so we know that older iterations are upgradable and default to correct vals.

Reply:
Thanks for the pick up! forgot to revert from an earlier test